packages/api/src/crud/courses/courses.controller.ts

```diff
@@ -2,8 +2,8 @@ import { Body, Controller, Get, Post, UseGuards } from '@nestjs/common';
 
 import { CourseBody, CreateCourseResponse, GetAllCoursesResponse } from '@coderscamp/shared/models/course';
 
-import { JwtAuthGuard } from '../auth/jwt/jwt-auth.guard';
 import { ForAdmin, RolesGuard } from '../auth/guards';
+import { JwtAuthGuard } from '../auth/jwt/jwt-auth.guard';
```

Comment:
💭 We can use `@/auth` here

Reply:
Not @/crud/auth if we want to have crud TS path?